rubi/rubi/data/market.py

```diff
@@ -77,7 +91,7 @@ def get_offers(
         maker: Optional[str] = None,
         from_address: Optional[str] = None,
         pair_name: Optional[str] = None,
-        book_side: Optional[OrderSide] = None,
+        book_side: Optional[OrderSide] = OrderSide.NEUTRAL,
```

Comment:
If this parameter needs to be supplied it should probably not be `Optional`, instead just: 

```python
book_side: OrderSide = OrderSide.NEUTRAL,
```